packages/paste-core/components/form-pill-group/src/FormPillGroup.tsx

```diff
@@ -39,26 +45,50 @@ export interface FormPillGroupProps
    * @memberof FormPillGroupProps
    */
   display?: "flex" | "inline-flex";
+  size?: FormPillGroupSizeVariant;
```

Comment:
suggestion(blocking): let's add a TSDoc comment for this new `size` prop like we do here for the other props. description can be something along the lines of what you wrote in the PR description which is super clear and informative. that way consumers can also easily see what the default size is when they look at our documentation for this component.

Reply:
Done.